### PR TITLE
Upgrade to Bootstrap 5

### PR DESCRIPTION
**Pull request by Mister-42**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  | -

Requires ~~https://github.com/yiisoft/yii-bootstrap5/pull/21~~

### CODE REVIEW

**Comment by samdark** [comment]
https://github.com/yiisoft/yii-bootstrap5/pull/21 merged.

File: public/js/app.js [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+    xhr.open('GET', event.target.href);
+    xhr.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
+    xhr.send();
+    xhr.onreadystatechange = function() {
+        if (xhr.readyState === 4) {
+            if(xhr.status === 200) {
+                document.querySelector(".load-more-comment-container").style.display = "none";
+                document.querySelector(".comment-feed-container").insertAdjacentHTML('beforeend', xhr.responseText);
+            } else {
+                document.querySelector(".comment-feed-container").innerHTML += 'An error occurred during your request: ' +  xhr.status + ' ' + xhr.statusText;
```
Comment (samdark) [inline public/js/app.js]:
```suggestion
                document.querySelector(".comment-feed-container").innerText += 'An error occurred during your request: ' +  xhr.status + ' ' + xhr.statusText;
```

File: public/js/app.js [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
+
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', event.target.href);
+    xhr.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
+    xhr.send();
+    xhr.onreadystatechange = function() {
+        if (xhr.readyState === 4) {
+            if(xhr.status === 200) {
+                event.target.parentNode.parentNode.remove();
+                document.querySelector(".comment-feed-container").insertAdjacentHTML('beforeend', xhr.responseText);
```
Comment (samdark) [inline public/js/app.js:15]:
```suggestion
                document.querySelector(".comment-feed-container").insertAdjacentText('beforeend', xhr.responseText);
```

File: public/js/app.js [hunk 3]
```diff
@@ -1,10 +1,24 @@
...
+            } else {
+                event.target.disabled = false;
+                document.querySelector(".comment-feed-container").insertAdjacentHTML('beforeend', 'An error occurred during your request: ' +  xhr.status + ' ' + xhr.statusText);
```
Comment (samdark) [inline public/js/app.js:18]:
```suggestion
                document.querySelector(".comment-feed-container").insertAdjacentText('beforeend', 'An error occurred during your request: ' +  xhr.status + ' ' + xhr.statusText);
```

**Comment by Mister-42** [comment]
Thanks!
I believe I got rid of all jQuery requirements and jQuery itself is not loaded in anymore. Comments loading as expected, carousel works, forms look good.

File: public/js/app.js (hunk 1 above)
Comment (Mister-42) [reply public/js/app.js]: What would be the benefit of treating the HTML we retrieved as text?

File: public/js/app.js (hunk 2 above)
Comment (Mister-42) [reply public/js/app.js:15]: What would be the benefit of treating the HTML we retrieved as text?

File: public/js/app.js (hunk 3 above)
Comment (Mister-42) [reply public/js/app.js:18]: This is text indeed, I will apply :)

File: public/js/app.js (hunk 2 above)
Comment (samdark) [reply public/js/app.js:15]: Less possibiltiy of having unexpected XSS>

Comment (samdark) [reply public/js/app.js:15]: Ah, in this case there's no sense in replacement. Good as is.

**Comment by samdark** [comment]
Any idea why it fails Codeception tests?

**Comment by Mister-42** [comment]
It's complaining about the form not being able to submit.

**Comment by Mister-42** [comment]
I used grep on `src` and forgot about `views` 🤦🏻

**Comment by Mister-42** [comment]
Adding/removing tags does not seem to work.

**Comment by Mister-42** [comment]
image

**Comment by samdark** [comment]
👍

**Pull request merged**